lld/test/COFF/lto-imp-prefix.ll

```diff
@@ -3,9 +3,11 @@
 ; RUN: rm -rf %t.dir
 ; RUN: split-file %s %t.dir
 ; RUN: llvm-as %t.dir/main.ll -o %t.main.obj
-; RUN: llvm-as %t.dir/other.ll -o %t.other.obj
+; RUN: llvm-as %t.dir/other1.ll -o %t.other1.obj
+; RUN: llvm-as %t.dir/other2.ll -o %t.other2.obj
 
-; RUN: lld-link /entry:entry %t.main.obj %t.other.obj /out:%t.exe /subsystem:console
+; RUN: lld-link /entry:entry %t.main.obj %t.other1.obj /out:%t.exe /subsystem:console
```

Comment:
I think you can define a `__imp_other` and link with `-debug:symtab` and check that `__imp_other` (not strictly needed) is emitted.

Reply:
Hmm, as an example of the unnecessary symbols that the implementation of #70777 does leave behind? That sounds like a good idea - although perhaps I should add that in a separate commit and not in this one.